main.js

```diff
@@ -0,0 +1,44 @@
+const showMobileMenu = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-menu');
+const crossDiv = document.querySelector('.close-menu');
+
+// On icon click:
+//  1) Hide the icon
+//  2) Show the mobile-menu  (open class)
```

Comment:
- [ ] Kindly remove the commented code as per best HTML practices.

Reply:
There are comment to clarify tle process the algorithm. They are used in programming as a tool to document.
This an example of zillions:
[https://www.geeksforgeeks.org/javascript-program-to-check-whether-two-strings-are-anagram-of-each-other/](url)